packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1005,6 +1005,10 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[mediaType][sendrecvType].headerBytesSent = result.headerBytesSent;
       this.statsResults[mediaType][sendrecvType].requestedBitrate = result.requestedBitrate;
       this.statsResults[mediaType][sendrecvType].requestedFrameSize = result.requestedFrameSize;
+
+      if (result.effect) {
```

Comment:
This `if` statement means if I turn off BNR/VBG, `this.statsResults[mediaType][sendrecvType].effect` will not be updated right? Is that what we want here?

Reply:
Yes, because the stat should be last used effect in this interval, so when we turn off the effect, we don't want to clear the statistic

Reply:
But that's right I missed cleaning part after interval, fixed with [this commit](https://github.com/webex/webex-js-sdk/pull/3619/commits/7e88b5200506bbe6f64a21a254bc1c9ad3c78a33)